src/devices/hilux.ts

```diff
@@ -9,6 +9,13 @@ const definitions: Definition[] = [
         description: 'Spot 7W',
         extend: [light({colorTemp: {range: [153, 370]}, powerOnBehavior: false})],
     },
+    {
+        zigbeeModel: ['Dimmer-Switch-ZB3.0'],
```

Comment:
Could you provide the data/database.db of this device? There are multiple devices which use the model ID so we should use a `fingerprint` here (example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/candeo.ts#L13)

Reply:
Is https://github.com/Koenkk/zigbee-herdsman-converters/pull/7763/commits/19e83eb3af83cc6f3267b6f0003fe21c6cff09c5 what you are asking for?

Reply:
Yes, can you also try if it works after removing `zigbeeModel: ['Dimmer-Switch-ZB3.0'],`?

Reply:
Just to make sure we are on the same page.
What I am testing is 

```
external_converters:
  - hilux.js
```
in configuration.yaml

hilux.js:
```
const {light} = require('zigbee-herdsman-converters/lib/modernExtend');

const definition = {
    fingerprint: [{modelID: 'Dimmer-Switch-ZB3.0', manufacturerName: 'Hilux'}],
    model: 'Dimmer-Switch-ZB3.0',
    vendor: 'Hilux',
    description: 'Zigbee LED dimmer smart switch',
    extend: [light({configureReporting: true})],
};

module.exports = definition;
```

Reply:
Perfect, thanks!